### PR TITLE
Sync dataset email redesign

### LINKED ISSUE #8937: Review sync maximum number of retries issue email
screen shot 2016-07-18 at 12 10 36

Part of #8889.
cc @urbanphes

### PR DESCRIPTION
**Pull request by MariaCheca**
Closes #8937 

screen shot 2016-08-08 at 16 15 51

CR @matallo

### CODE REVIEW

File: app/views/sync_mailer/max_retries_reached.html.erb [hunk 1]
```diff
@@ -1,64 +1,13 @@
...
-    </td>
-  </tr>
-<% end %>
+<%= render partial: 'mailer_modules/warning_message', locals: { message: message } %>
+
+<tr>
+  <td align="left" style="font-size: 14px; line-height: 25px; font-family: 'Open Sans', Helvetica, Arial, sans-serif; color: #647083;">
+    <p>The automatic synchronization of your dataset <strong><%=@dataset_name %></strong> has reached the maximum number of synchronization attempts and it's still failing to sync.</p>
+    <p>If you'd like to continue trying, log in to your account and manually reactivate synching by clicking “Sync now”</p>
+    <p>For more information about error messages and possible solution, <a href="https://docs.carto.com/cartodb-editor.html#error-codes-and-solutions" target="_blank" style="color: #1181FB;">click here</a>.</p>
```
Comment (matallo) [inline app/views/sync_mailer/max_retries_reached.html.erb]: wrong url https://carto.com/docs/carto-engine/import-api/import-errors/

**Comment by matallo** [comment]
neat! fix the url tho

File: app/views/sync_mailer/max_retries_reached.html.erb (hunk 1 above)
Comment (MariaCheca) [inline app/views/sync_mailer/max_retries_reached.html.erb]: Thanks! I didn't see that one.

**Comment by MariaCheca** [comment]
✨

**Pull request merged**